### PR TITLE
feat(backend): refactored customers permissions

### PR DESCRIPTION
**Pull request by allynt**
Refactored customers permissions to allow some actions to be performed by managers _or_ members.  Also removed the special case that allowed all actions to be performed by site admins.

### CODE REVIEW

File: astrosat_users/views/views_customers.py [hunk 1]
```diff
@@ -256,7 +280,7 @@ class CustomerUserInviteView(CustomerUserViewMixin, generics.GenericAPIView):
     A special view just for re-sending invitations.
     """
 
-    permission_classes = [IsAuthenticated, IsAdminOrManager]
+    permission_classes = [IsAuthenticated, IsManagerPermission]
     serializer_class = CustomerUserSerializer
 
     def post(self, request, *args, **kwargs):
```
Comment (R2ZER0) [inline astrosat_users/views/views_customers.py:286]: Does this allow a manager to re-send an invite to a CustomerUser from any Customer, not just their own Customer(s)?

File: astrosat_users/views/views_customers.py [hunk 2]
```diff
@@ -271,7 +295,7 @@ class CustomerUserOnboardView(CustomerUserViewMixin, generics.GenericAPIView):
     A special view just for onboarding Users.
     """
 
-    permission_classes = [IsAuthenticated, IsAdminOrManager]
+    permission_classes = [IsAuthenticated, IsManagerPermission]
     serializer_class = CustomerUserSerializer
 
     def post(self, request, *args, **kwargs):
```
Comment (R2ZER0) [inline astrosat_users/views/views_customers.py:301]: Similarly, does this allow a manager to "onboard" (idk what that means exactly :) ) any CustomerUser?

File: astrosat_users/views/views_customers.py (hunk 1 above)
Comment (allynt) [reply astrosat_users/views/views_customers.py:286]: No, it checks that the user making the request is a manager of the customer for whom the request is being made.

File: astrosat_users/views/views_customers.py (hunk 2 above)
Comment (allynt) [reply astrosat_users/views/views_customers.py:301]: See above.

File: astrosat_users/views/views_customers.py (hunk 1 above)
Comment (R2ZER0) [reply astrosat_users/views/views_customers.py:286]: Okay cheers, that took me a while to understand :)

**Review by R2ZER0 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**